client/src/Main.elm

```diff
@@ -1898,12 +1876,52 @@ showOverhang bsa1_overhang =
             "F__G"
 
 
-stringToOverhang : String -> Maybe Overhang
-stringToOverhang strOverhang =
+showMaybeBsa1Overhang : Maybe Bsa1Overhang -> String
```

Comment:
Why not `Bsa1Overhang -> Maybe String` as before?

Reply:
For the level0 elements the  Bsa1Overhang is mandatory but for the backbones it isn't.
So I created a function for showing bsa1Overhangs in level0 elements:
```elm
showBsa1Overhang : Bsa1Overhang -> String
```
And a function for showing the bsa1Overhang in backbones:
```elm
showMaybeBsa1Overhang : Maybe Bsa1Overhang -> String
```